src/v1_0/rw/cfg.cpp

```diff
@@ -5,12 +5,12 @@ namespace abb :: rws :: v1_0 :: rw :: cfg
     void loadCFGFile(RWSClient& client, const FileResource& resource)
     {
         std::stringstream uri;
-        uri << Resources::RW_CFG << "/action=load";
+        uri << Resources::RW_CFG << "?action=load";
```

Comment:
How did it work before?

Reply:
No, it didn't work, that's why I changed it